### ISSUE TITLE
RuntimeError: Failed to get Image Editor. This is unexpected.

### ISSUE THREAD

**Issue by Andrej730** [matyalatte/Blender-DDS-Addon#8]
Blender 3.5, got this error importing texture below.

```
RuntimeError: Failed to get Image Editor. This is unexpected.
Traceback (most recent call last):
  File "\addons\blender_dds_addon\ui\import_dds.py", line 166, in execute_base
    import_dds(context, os.path.join(directory, file.name))
  File "\addons\blender_dds_addon\ui\import_dds.py", line 123, in import_dds
    space = get_image_editor_space(context)
  File "\addons\blender_dds_addon\ui\bpy_util.py", line 24, in get_image_editor_space
    raise RuntimeError('Failed to get Image Editor. This is unexpected.')
RuntimeError: Failed to get Image Editor. This is unexpected.
```

**Comment by Andrej730** [comment]
Turn out I didn't have image editor opened. Fixed in #9

**Comment by matyalatte** [comment]
Thanks for the report!
I'll merge it.
